thumbor/handlers/__init__.py

```diff
@@ -373,13 +373,20 @@ def _fetch(self, url, extension, callback):
         else:
             self.context.statsd_client.incr('storage.miss')
 
-            def handle_loader_loaded(buffer):
+            def handle_loader_loaded(response):
+                if response is None:
+                    callback(False, None)
+                    return
+
+                buffer = response.body
```

Comment:
@acaciocenteno i think this is not working with the other handlers and because of that lots of the tests fail on travis with `AttributeError: 'str' object has no attribute 'body'`

```
[0mERROR:tornado.access:500 GET /unsafe/filters:max_age(30)/image.jpg (127.0.0.1) 4.73ms
[0mERROR:thumbor:ERROR: Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/tornado/web.py", line 1320, in _stack_context_handle_exception
    raise_exc_info((type, value, traceback))
  File "/home/travis/virtualenv/python2.7_with_system_site_packages/local/lib/python2.7/site-packages/tornado/web.py", line 1503, in wrapper
    result = method(self, *args, **kwargs)
  File "/home/travis/build/thumbor/thumbor/thumbor/handlers/imaging.py", line 100, in get
    self.check_image(kw)
  File "/home/travis/build/thumbor/thumbor/thumbor/handlers/imaging.py", line 96, in check_image
    return self.execute_image_operations()
  File "/home/travis/build/thumbor/thumbor/thumbor/handlers/__init__.py", line 97, in execute_image_operations
    self.filters_runner.apply_filters(thumbor.filters.PHASE_PRE_LOAD, self.get_image)
  File "/home/travis/build/thumbor/thumbor/thumbor/filters/__init__.py", line 81, in apply_filters
    callback()
  File "/home/travis/build/thumbor/thumbor/thumbor/handlers/__init__.py", line 122, in get_image
    self._fetch(self.context.request.image_url, self.context.request.extension, callback)
  File "/home/travis/build/thumbor/thumbor/thumbor/handlers/__init__.py", line 415, in _fetch
    self.context.modules.loader.load(self.context, url, handle_loader_loaded)
  File "/home/travis/build/thumbor/thumbor/thumbor/loaders/file_loader.py", line 22, in load
    callback(f.read())
  File "/home/travis/build/thumbor/thumbor/thumbor/handlers/__init__.py", line 381, in handle_loader_loaded
    buffer = response.body
AttributeError: 'str' object has no attribute 'body'
```